tox.ini

```diff
@@ -1,11 +1,15 @@
 [tox]
-envlist = py39
+envlist = py
 tox_pip_extensions_ext_venv_update = true
 tox_pip_extensions_ext_pip_custom_platform = true
 
 [testenv]
+description = Run unittests and linters
```

Comment:
description is not needed

Reply:
Doing a `tox -lav` would not give good output in absence of a description, especially as the setup of this repository is bit "unorthodox", as it combines linters with unittests into a single env. Still, it was not up to me to challange that decision ;)